components/actions/ActionListFilters.tsx

```diff
@@ -168,6 +168,12 @@ const MainCategory = styled.div`
   }
 `;
 
+const FilterColumn = styled(Col)`
```

Comment:
Juha shared a nice [styleguide](https://docs.twenty.com/developer/frontend/style-guide#styling) that recommends using the `Styled` prefix i.e. `StyledFilterColumn` to easily differentiate between styled components and _actual_ components. I kinda dig it, should we start following that convention?